### PR TITLE
Expose the Description field in the MSFT_AADApplication DSC resource

### PR DESCRIPTION
**Pull request by adhodgson1**
#### Pull Request (PR) description
This PR Exposes the Description field in the MSFT_AADApplication DSC resource

#### This Pull Request (PR) fixes the following issues
None

### CODE REVIEW

File: Tests/Integration/Microsoft365DSC/M365DSCIntegration.AAD.Create.Tests.ps1 [hunk 1]
```diff
@@ -32,6 +32,7 @@
                 {
                     DisplayName               = "AppDisplayName"
                     AvailableToOtherTenants   = $false
+                    Description                   = 'Test Description'
```
Comment (ykuijs) [inline Tests/Integration/Microsoft365DSC/M365DSCIntegration.AAD.Create.Tests.ps1:35]: Can you fix the indentation of the equals sign to match the other lines?

File: Tests/Unit/Microsoft365DSC/Microsoft365DSC.AADApplication.Tests.ps1 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [CmdletBinding()]
-param(
+                                                param(
```
Comment (ykuijs) [inline Tests/Unit/Microsoft365DSC/Microsoft365DSC.AADApplication.Tests.ps1:2]: Please remove the added spaces, so the param keyword remains at the beginning of the line.

File: docs/docs/resources/azure-ad/AADApplication.md [hunk 3]
```diff
@@ -8,6 +8,7 @@
 | **ObjectId** | Write | String | ObjectID of the app. | |
 | **AppId** | Write | String | AppId for the app. | |
 | **AvailableToOtherTenants** | Write | Boolean | Indicates whether this application is available in other tenants. | |
+| **Description** | Write | String | A free text field to provide a description of the application object to end users. The maximum allowed size is 1024 characters. | |
```
Comment (ykuijs) [inline docs/docs/resources/azure-ad/AADApplication.md:11]: We are generating the documentation during release. You can leave it for now, but next time you can save you the trouble 😉

**Review by ykuijs — CHANGES_REQUESTED** [review]
Great work. Could you please fix the two minor indentation issues?

**Comment by ykuijs** [comment]
And please add an entry in the changelog to reflect this change

File: Tests/Integration/Microsoft365DSC/M365DSCIntegration.AAD.Create.Tests.ps1 (hunk 1 above)
Comment (adhodgson1) [reply Tests/Integration/Microsoft365DSC/M365DSCIntegration.AAD.Create.Tests.ps1]: Sorry for those indentation issues. Both sorted.

**Review by ykuijs — APPROVED** [review]
LGTM

**Pull request merged**